litmus/PTX/CoRW-W.litmus

```diff
@@ -1,5 +1,5 @@
-PTX SB-strong
-"Fence-sc needed to prevent SB"
+PTX CoRW-W
```

Comment:
Should it be `CoRW-WW`, i.e. there are 2 writes in the second thread

Reply:
updated